cpp/src/converters/nvgraph.cu

```diff
@@ -43,9 +43,8 @@ void createGraph_nvgraph(nvgraphHandle_t nvg_handle,
   // setup nvgraph variables
   if (use_transposed) {
     // convert edgeList to transposedAdjList
-    if (gdf_G->transposedAdjList == nullptr) {
-      cugraph::add_transposed_adj_list(gdf_G);
-    }
+    CUGRAPH_EXPECTS(gdf_G->transposedAdjList != nullptr,
+              "Invalid API parameter");
```

Comment:
I'm not sure this error message makes sense.  It does not tell the user which parameter is causing the error.
Maybe something like:  "Invalid API parameter - transposed data is missing"

Reply:
This is following the errors we are throwing in other algorithms currently. 
Better error messages could be a completely separate PR with changes for all such ambiguities.